### DISCUSSION TITLE
Ansible- Load Schema Catalogue

### DISCUSSION THREAD

**Discussion by aravindkumar1982** [learndevopsonline/learndevopsonline#1400] [Q&A]
Hello @r-devops ,

Hope you are all well, I have been trying to run the catalogue componet via Ansible. I am getting the following error. Please note I did cross-check the code. I am not sure what the issue.  Please note servers are up and running.

![Screenshot 2023-09-01 at 17 32 11](https://github.com/learndevopsonline/learndevopsonline/assets/139839369/301e690c-ffe3-439d-9f3c-aa3e62b3a00e)

Hope this helps.

Thanks,

Aravind

**Comment by sreedharm07** [comment]
hi @aravindkumar1982 ,
to call the variable in shell we use $.
but in ansible no need to use $ to call variable , just flower breshaes enough {{component}}.
remove $.

![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/c51576aa-6367-44e1-8121-09d548eca645)

**Reply by aravindkumar1982** [reply]
@r-devops Thanks a lot. It did work. Sorry for the trouble.
